js/app/access-ucl-eligibility-widget.js

```diff
@@ -368,10 +373,10 @@
                 // End event handling.
                 
                 // HTML template
-var widget = '<p>Enter your details below to check your eligibility ' + text2 + ' (you will need your home postcode and the postcode of the school where you took your A levels).</p>\
+/*var widget = '<p>Enter your details below to check your eligibility ' + text2 + ' (you will need your home postcode and the postcode of the school where you took your A levels).</p>\
 <p>Please bear in mind that the results given here are indicative only.</p>\
-\
-<form action="" method="get" id="checker-widget" autocomplete="off">\
+\*/
+var widget = '<form action="" method="get" id="checker-widget" autocomplete="off">\
```

Comment:
Is the single quote mark supposed to be there?

Reply:
Yep. That's a JS template that is.